src/DssProxy.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 // Copyright (C) 2022 Dai Foundation
+// Copyright (C) 2017  DappHub, LLC
```

Comment:
Can you move the DappHub line above the Dai Foundation so that they are in chronological order?

Bonus points if you fix the spacing.